README.md

```diff
@@ -1,56 +1,65 @@
 # zkLLVM
 
 zkLLVM is a compiler from high-level programming languages into an input for provable computations protocols. 
-It can be used to generate input for any arbitrary zero-knowledge proof system or protocol, which accepts 
-input data in form of algebraic circuits It assumed to be used together with `Placeholder` proof system or 
-any other arithmetization compatible with `Placeholder` proof system.
+It can generate input for any arbitrary zero-knowledge proof system or protocol, which accepts 
+input data in form of algebraic circuits. It should be used together with `Placeholder` proof system or 
```

Comment:
The `Placeholder` will soon be replaced with something real, right? 

![image](https://user-images.githubusercontent.com/8015154/211201926-df9cc5cd-31f6-47b3-a368-a6081ee1d61b.png)

Reply:
Nope, this is the actual proof system name :)

Reply:
Okay, I'll make it a link )